W/WCPG/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WCPG"
+version = v"0.9.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/fixif/WCPG/archive/refs/tags/0.9.tar.gz", "4f6b1d2abc298891ae9e3966428c2d8b4e8bbc3528e917d261d8613dea41ab7d"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/libblas.${dlext}
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/liblapack.${dlext}
```

Comment:
Please don't create symlinks like this.  Doesn't the build system allow to specify the blas library to link to?

Reply:
Currently it is not possible to specify the blas library to link to. Instead of working with more patches we will modify WCPG directly to allow it.